torchgeo/datasets/idtrees.py

```diff
@@ -612,10 +613,10 @@ def plot_las(self, index: int, colormap: Optional[str] = None) -> Any:
             else:
                 colors = np.zeros_like(points)
 
-        pcd = open3d.geometry.PointCloud()
-        pcd.points = open3d.utility.Vector3dVector(points)
-        pcd.colors = open3d.utility.Vector3dVector(colors)
-        vis = open3d.visualization.Visualizer()
-        vis.create_window()
-        vis.add_geometry(pcd)
-        return vis
+        point_cloud = pyvista.PolyData(points)  # type: ignore[attr-defined]
+        point_cloud["color"] = colors
```

Comment:
I'd recommend a `to_pyvista()` method to let users easily get a PyVista PolyData object without plotting for them